frontend/src/components/module-viewer/content/ContentBlock.tsx

```diff
@@ -48,6 +48,7 @@ const ContentBlock = ({
     onClose: onDeleteModalClose,
   } = useDisclosure();
   const context = useContext(EditorContext);
+  const isColumn = block.type.clientType !== "column";
```

Comment:
Shouldn't this be `block.type.clientType === "column"` based on the name?

Reply:
Good catch